mbzirc_ign/src/GameLogicPlugin.cc

```diff
@@ -755,13 +813,19 @@ void GameLogicPlugin::PreUpdate(const UpdateInfo &_info,
   // give time penalties or disable robots if they exceeded
   // boundaries
   this->dataPtr->CheckRobotsInGeofenceBoundary(_ecm);
+
+  // disable objects dropped into the ocean
+  this->dataPtr->DisableDroppedObjects(_ecm);
 }
 
 //////////////////////////////////////////////////
 void GameLogicPlugin::PostUpdate(
     const ignition::gazebo::UpdateInfo &_info,
     const ignition::gazebo::EntityComponentManager &_ecm)
 {
+//  if (_info.paused)
```

Comment:
Just checking, is this on purpose?

Reply:
yeah I noticed that scoring file won't be updated if paused, e.g. after competition ended. Removed commented out code in 9ba0c341b316cb4440b5d681f04977ae1de6eb60